### PR TITLE
Fixownerref4referredresouce

### PR DESCRIPTION
**Pull request by ianzhang366**

### CODE REVIEW

File: pkg/controller/subscription/reference.go [hunk 1]
```diff
@@ -32,9 +33,10 @@ import (
 
 var SecretKindStr = "Secret"
 var ConfigMapKindStr = "ConfigMap"
+var SubscriptionGVK = schema.GroupVersionKind{Group: "", Kind: "Subscription", Version: "v1alpha1"}
 
 //SercertReferredMarker is used as a label key to filter out the secert coming from reference
-var SercertReferredMarker = "IsReferredBySub"
+var SercertReferredMarker = "IsReferredBySub-"
 
 type referredObject interface {
```
Comment (kuanf) [inline pkg/controller/subscription/reference.go]: Actually, you can do this to include 2 interface into 1
```
type referredObject interface {
	runtime.Object
	metav1.Object
}
```

**Comment by kuanf** [comment]
why do you always have sooooooooooo many commits?  something wrong with your personal repo?

File: pkg/controller/subscription/reference.go (hunk 1 above)
Comment (ianzhang366) [reply pkg/controller/subscription/reference.go]: fixed this.

**Pull request merged**